native/src/components/OpeningHours.tsx

```diff
@@ -62,7 +62,7 @@ const OpeningHours = ({
   const openingHoursTitle = (
     <TitleContainer language={language}>
       <Text style={{ fontWeight: 'bold', alignSelf: 'center' }}>{t('openingHours')}</Text>
-      <OpeningLabel isOpened={isCurrentlyOpen} direction={contentDirection(language)}>
+      <OpeningLabel isOpened={isCurrentlyOpen} contentDirection={contentDirection(language)}>
```

Comment:
Apparently, a `Text` can throw when it has a property `direction`. https://stackoverflow.com/questions/60479653/invalid-ygdirection-row-should-be-one-of-inherit-ltr-rtl-react-native